docs/getting-started-guides/README.md

```diff
@@ -6,9 +6,9 @@ For the easiest "kick the tires" experience, please try the [local docker](docke
 If you are considering contributing a new guide, please read the
 [guidelines](../../docs/devel/writing-a-getting-started-guide.md).
 
-IaaS Provider        | Config. Mgmt | OS     | Networking  | Docs                                                                           | Conformance | Support Level                | Notes
+IaaS Provider        | Config. Mgmt | OS     | Networking  | Docs                                                                           | Conforms    | Support Level                | Notes
 -------------------- | ------------ | ------ | ----------  | ------------------------------------------------------------------------------ | ----------- | ---------------------------- | -----
-GKE                  |              |        | GCE         | [docs](https://cloud.google.com/container-engine)                              |             | Commercial                   | Uses K8s version 0.15.0
+GKE                  |              |        | GCE         | [docs](https://cloud.google.com/container-engine)                              | [✓](1)      | Commercial                   | Uses K8s version 0.15.0
```

Comment:
1. so that the column just reads as either a checkmark or not.  More visual impact
2. to limit column width so it fits on screen, both rendered, and in text mode.